app-containers/syft/syft-0.51.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module
+
+DESCRIPTION="Generate a Software Bill of Materials from container images and filesystems"
+HOMEPAGE="https://www.anchore.com"
+SRC_URI="https://github.com/anchore/syft/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://dev.gentoo.org/~williamh/dist/${P}-deps.tar.xz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+# Flatcar: keyword for arm64 and amd64
+KEYWORDS="amd64 arm64"
+
+src_compile() {
+	# Flatcar: add ldflags to set version
```

Comment:
Thanks! I think this change can be upstreamed with the `dobin bin/*` indentation fix. We should also submit a keyword request (`~arm64` or `arm64`)

Reply:
I've opened a PR here: https://github.com/gentoo/gentoo/pull/26617